src/event/event.php

```diff
@@ -186,12 +194,117 @@ private static function pixChargebackLogResource()
         };
     }
 
+    /**
+    # Retrieve a specific notification Event
+
+    Receive a single notification Event object previously created in the Stark Infra API by passing its id
+
+    ## Parameters (required):
+        - id [string]: object unique id. ex: "5656565656565656"
+
+    ## Parameters (optional):
+        - user [Organization/Project object, default null]: Organization or Project object. Not necessary if StarkInfra\Settings::setUser() was used before function call
+
+    ## Return:
+        - Event object with updated attributes
+     */
+    public static function get($id, $user = null)
+    {
+        return Rest::getId($user, Event::resource(), $id);
+    }
+
+    /**
+    # Retrieve notification Events
+
+    Receive a enumerator of notification Event objects previously created in the Stark Infra API
```

Comment:
trocar para:

 Receive a enumerator of Event objects previously created in the Stark Infra API